docs/markdown/android/tabs.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Tabs
+description: Tabs may be used navigate between multiple URLs. Tabs are intended as page-level navigation.
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/5b/b9/0e/5bb90ee7b0b2e9f336cb803303f3eecf.png" width={413} height={48} alt="an example of tabs"/>
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="When to use" />
+      - To break up a large collection of content into logical, digestible views.
+      - To switch between different, yet related views, such as Updates and Messages.
+  </Group>
+  <Group>
+  <Dont title="When not to use" />
+    - When any UI or content above the Tabs is altered upon selection. Use Link instead. 
+    - To break up content that is not related to each other or is not on the same hierarchical level. 
+    - To filter content. Consider SegmentedControl instead.
+  </Group>
+</TwoCol>
+
+## Best practices
+
+- Place Tabs directly above the relevant content. 
+- Avoid using more than 6 Tabs. Consider using a different component like a Sheet.
+- Order Tabs by relevance - the first tab should be the most logical starting view.
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/d7/27/63/d727634455161d98e7614c8b12b956f6.png" width={233} height={48} alt="example with concise tab labels"/>
+    <Do title="Do" />
+    Keep Tab labels short and concise, one or two words.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/ae/f7/f8/aef7f8343979091c40c01111e732a95c.png" width={78} height={48} alt="example of only one tab"/>
+    <Dont title="Don't" />
+    Hide or disable Tabs if that Tab’s content is empty. Additionally, there should always be at least two tabs.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/0d/08/71/0d08712e4810341422d3d20fbe9fd4cb.png" alt="example of tabs scrolling horizontally"/>
+    <Do title="Do" />
+    Allow tabs to scroll horizontally.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/fe/97/b2/fe97b2a279741762771f98bcd47e8e15.png" alt="example of tabs with truncated labels"/>
+    <Dont title="Don't" />
+    Truncate tabs to fit within a view.
+  </Group>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/09/e9/b6/09e9b68dea0327b92311472655e416a1.png" alt="example of using too many tabs"/>
+    <Dont title="Don't" />
+    Use more than 6 Tabs. Consider using a different component like a Sheet.
+  </Group>
```

Comment:
given that this is already mentioned above in the bullet points, and it doesn't have a Do match, can we delete it?

Reply:
I'm down. Any concerns @khalvy?

Reply:
Yea, sounds good to me!

Reply:
Done!